examples/demo-stf/src/batch_builder.rs

```diff
@@ -89,6 +89,7 @@ where
 
             // Execute
             {
+                // TODO: Bug(!), because potential discrepancy. Should be resolved by https://github.com/Sovereign-Labs/sovereign-sdk/pull/361
```

Comment:
```suggestion
                // TODO: Bug(!), because potential discrepancy. Should be resolved by https://github.com/Sovereign-Labs/sovereign-sdk/issues/434
```